recipe/meta.yaml

```
@@ -37,31 +29,34 @@ requirements:
     - python
     - pip
     - setuptools
-    - mysql {{ version }}  # [not win]
-    - protobuf 5.29.3
-    - libprotobuf 5.29.3
+    - mysql-devel {{ version }}
+    - mysql-client {{ version }}
   run:
     - python
-    - mysql  # [not win]
-    - protobuf
   run_constrained:
-    - dnspython >=1.16.0,<=2.3.0
-    - python-gssapi >=1.6.9,<=1.8.2
-    - deprecated >=1.2.6
-    - typing-extensions >=3.7.4
-    - zipp >=0.5
+    - dnspython ==2.6.1
```

I think we don`t have to use strict pinning, it makes this pkg unistallable in many environments in that case.
Better just rewrite on:
```suggestion
    - dnspython >=2.6.1,<2.7.0 
```